lisa/features/sriov.py

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from lisa.feature import Feature
+
+FEATURE_NAME_SRIOV = "Sriov"
+
+
+class Sriov(Feature):
+    @classmethod
+    def name(cls) -> str:
+        return FEATURE_NAME_SRIOV
+
+    @classmethod
+    def enabled(cls) -> bool:
```

Comment:
This should be implemented to detect if sriov is enabled on current node. Currently, it's a classmethod, but it looks should be an instance method. Please help on removing the class decorator in a separated commit.